notifications/utils.py

```diff
@@ -24,8 +24,10 @@ def send_notification(email, notification_type, context=None, language=DEFAULT_L
     if context is None:
         context = {}
 
+    template = NotificationTemplate.objects.get(type=notification_type)
```

Comment:
This will throw if there no `NotificationTemplate` object at all, which wasn't the case before, and IMO is not optimal.

Reply:
ah, right! missed that when refactoring